.github/workflows/publish_upon_merge.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and publish conda packages upon merge
+on:
+  # Trigger the workflow on pull request, but only for the main branch.
+  # By default, a workflow only runs when a pull_request's activity type
+  # is opened, synchronize, or reopened. To trigger workflows for more activity types, use the types keyword.
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+    # explicit definition of the paths, probably unnecessary
+    paths:
+      - 'atlas-experiment-metadata/**'
+      - 'atlas-fastq-provider/**'
+      - 'atlas-gene-annotation-manipulation/**'
+      - 'cell-types-analysis/**'
+      - 'irap-bamutils/**'
+      - 'irap-components/**'
+      - 'r-atlas-internal/**'
+jobs:
+  publish:
+    # this job will only run if the PR has been merged
+    if: github.event.pull_request.merged == true
+    runs-on: [ubuntu-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: publish-to-conda
```

Comment:
My worry with this action is that the docs say it only supports selected channels- do you think this would work with our conda channel? https://github.com/marketplace/actions/publish-conda